Packages/MIES/MIES_RepeatedAcquisition.ipf

```diff
@@ -495,11 +507,32 @@ Function RA_SkipSweeps(panelTitle, skipCount, [limitToSetBorder])
 			followerPanelTitle = StringFromList(i, listOfFollowerDevices)
 			NVAR followerCount = $GetCount(followerPanelTitle)
 			followerCount = RA_SkipSweepCalc(followerPanelTitle, skipCount)
+
+			if(document)
+				RA_DocumentSweepSkipping(panelTitle, skipCount)
+			endif
+
 			RA_StepSweepsRemaining(followerPanelTitle)
 		endfor
 	endif
 End
 
+/// @brief Document the number of skipped sweeps
+static Function RA_DocumentSweepSkipping(string panelTitle, variable skipCount)
+
+	variable sweepNo
+
+	Make/FREE/N=(1, 1, LABNOTEBOOK_LAYER_COUNT) vals = NaN
+	vals[0][0][INDEP_HEADSTAGE] = skipCount
+	Make/T/FREE/N=(3, 1) keys
+	keys[0] = "Sweep Skipping"
```

Comment:
Let's call it 'skip sweeps' instead

Reply:
@timjarsky Done. I adjusted the capitalization though.